__tests__/storage/getStopGroupByGpuid.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { Storage } from '../../src/classes/storage';
+import { fakeGroundPlacesJSON } from '../../mocks/groundPlaces';
+
+describe('getStopGroup()', () => {
+  const StorageInstance: Storage = new Storage(fakeGroundPlacesJSON);
+
+  it('should return the right StopGroup based on its Gpuid', () => {
+    const getStopGroupByGpuid = StorageInstance.getStopGroupByGpuid('g|FRststbi__@u0tkxd');
+
+    expect(getStopGroupByGpuid).toEqual({
```

Comment:
=> strictEqual, and you can remove the const

Reply:
Done